src/dstack/_internal/server/models.py

```diff
@@ -188,15 +203,17 @@ class JobModel(BaseModel):
     submitted_at: Mapped[datetime] = mapped_column(DateTime)
     last_processed_at: Mapped[datetime] = mapped_column(DateTime)
     status: Mapped[JobStatus] = mapped_column(Enum(JobStatus))
-    error_code: Mapped[Optional[JobErrorCode]] = mapped_column(Enum(JobErrorCode))
+    termination_reason: Mapped[Optional[JobTerminationReason]] = mapped_column(
+        Enum(JobTerminationReason)
+    )
     job_spec_data: Mapped[str] = mapped_column(String(4000))
     job_provisioning_data: Mapped[Optional[str]] = mapped_column(String(4000))
     runner_timestamp: Mapped[Optional[int]] = mapped_column(Integer)
     # `removed` is used to ensure that the instance is killed after the job is finished
-    removed: Mapped[bool] = mapped_column(Boolean, default=False)
     remove_at: Mapped[Optional[datetime]] = mapped_column(DateTime)
     instance: Mapped[Optional["InstanceModel"]] = relationship(back_populates="job")
     used_instance_id: Mapped[Optional[uuid.UUID]] = mapped_column(UUIDType(binary=False))
+    replica_num: Mapped[int] = mapped_column(Integer)
```

Comment:
Maybe define replica_num closer to the top for better readability.

Reply:
I suggest reordering all fields as a separate issue. I don't see "the only best" solution here right now.